src/style.css

```diff
@@ -0,0 +1,86 @@
+html {
+  --rectangle-big-height: 70px;
+  --rectangle-small-height: 42px;
+
+  --big-image-height: 19px;
+  --small-image-height: 11px;
+
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search-image {
+  position: absolute;
+}
+
+.search-image--big {
+  height: var(--big-image-height);
+  top: calc((var(--rectangle-big-height) - var(--big-image-height)) / 2);
```

Comment:
it's better to use `%`

Reply:
![Code_lXcFrASorJ](https://user-images.githubusercontent.com/94491034/201599377-c3c4c7ea-4b5d-479c-81f5-2e4c60e3bfed.png)

Reply:
What am I doing wrong?